src/libsyntax/ast.rs

```diff
@@ -10,7 +10,7 @@ use crate::parse::token;
 use crate::print::pprust;
 use crate::ptr::P;
 use crate::source_map::{dummy_spanned, respan, Spanned};
-use crate::symbol::{kw, Symbol};
+use crate::symbol::{kw, Symbol, sym};
```

Comment:
```suggestion
use crate::symbol::{kw, sym, Symbol};
```
Style nit (here and in couple of other places).